tests/functional/generate_pages.py

```diff
@@ -194,11 +194,11 @@
 
 """
 
-LIST_SUMMARY_EDIT_LINK_GETTER = r"""  listEditLink(instance) { return `tbody:nth-child(${instance}) td:last-child a:first-child`; }
+LIST_SUMMARY_EDIT_LINK_GETTER = r"""  listEditLink(instance) { return `tbody:nth-child(${instance}) td:last-child a[aria-label^='Change'`; }
```

Comment:
This works great for now, though I would be happier still with something that wasn't subject to a potential content change (i.e. "could you make this say 'alter' instead?") Perhaps a template change to add a class we could select on, or a non user-visible attribute value of our own invention, such as eq-primary-person?

Reply:
I think the only other reasonable way to do it would be to add a data tag. We do this elsewhere, but this would require a change to the pattern library in this case I think.

Reply:
I hadn't known of the data-* attributes before, that would be a good way to do this, I like the side effects of populating the element.dataset in the DOM. Handy.

Reply:
Fixed with data attribute